src/adagrams.js

```diff
@@ -1,15 +1,135 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPool = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+  };
+
+  const keys = Object.keys(letterPool);
+  let letter;
+  const hand = [];
+  
+  while (hand.length < 10) {
+    letter = keys[Math.floor(Math.random() * keys.length)];
+    if (letterPool[letter] > 0) {
+      hand.push(letter);
+      letterPool[letter] -= 1;
+    }
+  }
```

Comment:
It also seems like y'all made ✨ Hard Mode ✨ as consonants (t, n, h, etc.) have the same statistical probability of appearing in your hand as vowels. This approach doesn't take into account the expected frequency for vowels like 'A' which should have a 12/98 probability of being picked (12 times more than 'Z'). Instead, we created a list of all 26 letters, each letter only appearing once, giving all letters an equal statistical probability of being picked (1/26).